docs/style-guide.adoc

```diff
@@ -35,7 +43,7 @@ ORDER BY score DESC
 
 == Rules
 
-Whenever two rules are in conflict, use the order to decide which one trumps.
+In case two rules are in conflict, and there is no explicit mention which rule trumps, the last mentioned rule applies.
```

Comment:
"In case two rules are in conflict, and there is no explicit mention which rule trumps, the last mentioned rule applies" -> "In case two rules are in conflict, and there is no explicit mention of which rule trumps, the rule mentioned last applies"